test/python/tools/monitor/test_backend_monitor.py

```diff
@@ -95,8 +95,8 @@ def test_backend_monitor(self, _):
 
         stdout = fake_stdout.getvalue()
         self.assertIn('Configuration', stdout)
-        self.assertIn('Qubits [Name / Freq / T1 / T2 / U1 err / U2 err / U3 err / Readout err]',
-                      stdout)
+        self.assertIn('Qubits [Name / Freq / T1 / T2 / ', stdout)
```

Comment:
Could a test be added for the new gates being added to the header?

Reply:
added in 36d1eae